oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -1015,7 +1018,7 @@ private String getConsoleUrl(String cluster, String actionId) throws FalconExcep
             // Get conf when workflow is launched.
             if (coordinatorAction.getExternalId() != null) {
                 WorkflowJob jobInfo = client.getJobInfo(coordinatorAction.getExternalId());
-
+                workflowUser = jobInfo.getUser();
```

Comment:
From the logs reported by @muraliramasami , the rerun user is falcon when the coordinator (thus the workflow) were by user "hrt_qa". So CurrentUser.authenticate(message.getWorkflowUser()) is supposed to work and run re-run as message.getWorkflowUser(), but it is not accurate anymore.  Let me debug some more.